README.md

```diff
@@ -1,85 +1,41 @@
 
 Included in this demo app:
 
-- two login clients
-- hooks for easy usage
+- hooks for easy usage with React
 - redux store listening a client
 - HOC component listening a client and showing different content for authorized and unauthorized users.
+- getting API token and using it to get Profile.
 
-Clients dispatch events and trigger changes which then trigger re-rendering of the components using clients.
-
-## using oidc-client
-
-Used client type is defined in .env.
-Settings for keycloak
-
-```bash
-REACT_APP_OIDC_CLIENT_TYPE="keycloak"
-REACT_APP_OIDC_SILENT_AUTH_PATH="/silent-check-sso.html"
-REACT_APP_OIDC_CALLBACK_PATH="/"
-```
-
-Settings for oidc-react
-
-```bash
-REACT_APP_OIDC_CLIENT_TYPE="oidc"
-REACT_APP_OIDC_SILENT_AUTH_PATH="/silent_renew.html"
-REACT_APP_OIDC_CALLBACK_PATH="/callback"
-```
-
-## Oidc and keyclock client differences
-
-Client libraries trigger different events when client status changes or an error occurs.
-
-CLIENT_READY:
-
-- oidc does not trigger this event. Keycloak triggers this when onReady() is called. This is same as either AUTHORIZE or UNAUTHORIZED event.
-  TOKEN_EXPIRING:
-- triggered only by oidc
-  ERROR event with type AUTH_ERROR:
-- Oidc trigger the event if silent signin results in error, but not if error is 'login_required'. Keycloak triggers this error when onAuthError() is called
+Client dispatches events and trigger changes which then trigger re-rendering of the components using the client.
 
 ## Config
 
-use .env -files. Some values are client specific. Default client is keycloak and relevant settings are:
-
-```bash
-REACT_APP_OIDC_URL="https://tunnistus.hel.ninja/auth"
-REACT_APP_OIDC_REALM="helsinki-tunnistus"
-REACT_APP_OIDC_CLIENT_ID="https://api.hel.fi/auth/example-ui-profile"
-```
+Configs are in .env -files. Default OIDC-server is Tunnistamo.
 
-other settings should not be changed
+Tunnistamo does not support silent login checks (it uses only localstorage) so REACT_APP_OIDC_AUTO_SIGN_IN must be 'false'. It renews access tokens so REACT_APP_OIDC_SILENT_AUTH_PATH must be changed to '/' to prevent errors for unknown redirect url.
 
 Config can also be overridden for command line:
 
 ```bash
 REACT_APP_OIDC_URL=https://foo.bar yarn start
 ```
 
-### Config for Tunnistamo
+### Config for Keycloak OIDC provider
+
+Settings when using keycloak server:
 
 ```bash
-REACT_APP_OIDC_URL="https://api.hel.fi/sso-test"
-REACT_APP_OIDC_CLIENT_ID="https://api.hel.fi/auth/helsinkiprofile-ui-dev"
-REACT_APP_OIDC_SILENT_AUTH_PATH="/callback"
-REACT_APP_OIDC_AUTO_SIGN_IN=false
-REACT_APP_OIDC_SCOPE="openid profile https://api.hel.fi/auth/helsinkiprofiledev"
-REACT_APP_OIDC_CLIENT_TYPE="oidc"
-REACT_APP_OIDC_REALM=""
-REACT_APP_OIDC_CALLBACK_PATH="/callback"
-REACT_APP_OIDC_TOKEN_EXCHANGE_PATH="/api-tokens/"
+REACT_APP_OIDC_URL="https://helsinki-profile-keycloak-dev.agw.arodevtest.hel.fi/auth"
```

Comment:
I wouldn't put this temporary URL here. Perhaps write "TODO" here or some imaginary URL like "https://your-keycloak-server.example/auth".

Reply:
Changed to <KEYCLOAK_SERVER_URL>/auth